Fix 8.15.5-1 build

legrego opened the pull request:
## Pull Request Overview

This PR fixes build issues for version 8.15.5-1 by updating artifact naming conventions, modifying download URLs for Beats dependencies, and updating Buildkite pipeline configurations to better support cloud deployments.  
- Update tarball names by removing "-SNAPSHOT" from the version strings.  
- Simplify the download process for Beats by making the build ID optional and switching to a direct download URL.  
- Modify Buildkite pipeline settings, including increased timeouts and added disk size configuration.

kc13greiner committed:
Hide the chrome popup about leaked passwords (#215414)

Closes https://github.com/elastic/kibana/issues/214355

Testing with Chrome 135.x.x.x resulted in a pop up that was breaking the
flow of tests:

<img width="470" alt="Screenshot 2025-03-20 at 4 25 17 PM"
src="https://github.com/user-attachments/assets/df908294-1881-4b6d-b9a2-3027b37b06ad"
/>

This isn't captured by the screenshotting plugin, but is visible if you
run `headless=0`

This adds the config to disable that popup since our generic testing
passwords are very common and therefor flagged

legrego committed:
Use a fixed version of the ES Snapshot Manifest

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306163)
* Commit: 51e06358fc8b8323237981f478596ad1e3e26be5

### Failed CI Steps
* [Pre-Build](https://buildkite.com/elastic/kibana-pull-request/builds/306163#019744f1-254b-49a0-94a2-ff1fe6fe8c61)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306163","number":306163,"commit":"51e06358fc8b8323237981f478596ad1e3e26be5"}],"number":306163}
buildkite-pr-comment-->

legrego committed:
Do not resolve bucket

legrego committed:
no-op

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306355)
* Commit: 75a535a742052bf4f71bbefa5c7d0da4516b99d9

### Failed CI Steps
* [Build Kibana Distribution and Plugins](https://buildkite.com/elastic/kibana-pull-request/builds/306355#0197460c-be54-42db-a94e-a06ab7c6c5c1)

### History
* :broken_heart: [Build #306209](https://buildkite.com/elastic/kibana-pull-request/builds/306209) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306169](https://buildkite.com/elastic/kibana-pull-request/builds/306169) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306163](https://buildkite.com/elastic/kibana-pull-request/builds/306163) failed 51e06358fc8b8323237981f478596ad1e3e26be5

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306355","number":306355,"commit":"75a535a742052bf4f71bbefa5c7d0da4516b99d9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306209","number":306209,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306169","number":306169,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306163","number":306163,"commit":"51e06358fc8b8323237981f478596ad1e3e26be5"}],"number":306355}
buildkite-pr-comment-->

legrego committed:
no-op again

legrego committed:
no-op 3

jbudz committed:
[build] Download released beats

legrego committed:
no-op 4

jbudz committed:
increase disk

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306396)
* Commit: 083fa1c326f332317b3a6acc0fbd6d7bd7703c33

### Failed CI Steps
* [Build Kibana Distribution and Plugins](https://buildkite.com/elastic/kibana-pull-request/builds/306396#01974670-480d-4bd4-8209-a8e6397e6899)

### History
* :broken_heart: [Build #306355](https://buildkite.com/elastic/kibana-pull-request/builds/306355) failed 75a535a742052bf4f71bbefa5c7d0da4516b99d9
* :broken_heart: [Build #306209](https://buildkite.com/elastic/kibana-pull-request/builds/306209) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306169](https://buildkite.com/elastic/kibana-pull-request/builds/306169) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306163](https://buildkite.com/elastic/kibana-pull-request/builds/306163) failed 51e06358fc8b8323237981f478596ad1e3e26be5

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306396","number":306396,"commit":"083fa1c326f332317b3a6acc0fbd6d7bd7703c33"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306355","number":306355,"commit":"75a535a742052bf4f71bbefa5c7d0da4516b99d9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306209","number":306209,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306169","number":306169,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306163","number":306163,"commit":"51e06358fc8b8323237981f478596ad1e3e26be5"}],"number":306396}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306413)
* Commit: 570ddcfef71b2a20a401c37773dda7b04ed71d91

### Failed CI Steps
* [Build and Deploy to Cloud](https://buildkite.com/elastic/kibana-pull-request/builds/306413#019746a4-3e70-4f65-a78b-3e4847fed387)

### History
* :broken_heart: [Build #306396](https://buildkite.com/elastic/kibana-pull-request/builds/306396) failed 083fa1c326f332317b3a6acc0fbd6d7bd7703c33
* :broken_heart: [Build #306355](https://buildkite.com/elastic/kibana-pull-request/builds/306355) failed 75a535a742052bf4f71bbefa5c7d0da4516b99d9
* :broken_heart: [Build #306209](https://buildkite.com/elastic/kibana-pull-request/builds/306209) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306169](https://buildkite.com/elastic/kibana-pull-request/builds/306169) failed b775d25b8c700b5c1a9def9c44432d178a61e800
* :broken_heart: [Build #306163](https://buildkite.com/elastic/kibana-pull-request/builds/306163) failed 51e06358fc8b8323237981f478596ad1e3e26be5

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306413","number":306413,"commit":"570ddcfef71b2a20a401c37773dda7b04ed71d91"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306396","number":306396,"commit":"083fa1c326f332317b3a6acc0fbd6d7bd7703c33"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306355","number":306355,"commit":"75a535a742052bf4f71bbefa5c7d0da4516b99d9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306209","number":306209,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306169","number":306169,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306163","number":306163,"commit":"51e06358fc8b8323237981f478596ad1e3e26be5"}],"number":306413}
buildkite-pr-comment-->

jbudz committed:
[cloud deploy] skip es image override

Instead of using our functional test verified image (that's out of date) this uses the default cloud ES image.

jbudz committed:
fix unset

jbudz committed:
increase timeout

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306462)
* Commit: 893601577dd3f1f35c7972ab4ed47c4aeffce039
* [Cloud Deployment](https://kibana-pr-222966.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #306428](https://buildkite.com/elastic/kibana-pull-request/builds/306428) failed 582be89666777c946819d3ed5c3c5da5f3cdf692
* :broken_heart: [Build #306413](https://buildkite.com/elastic/kibana-pull-request/builds/306413) failed 570ddcfef71b2a20a401c37773dda7b04ed71d91
* :broken_heart: [Build #306396](https://buildkite.com/elastic/kibana-pull-request/builds/306396) failed 083fa1c326f332317b3a6acc0fbd6d7bd7703c33
* :broken_heart: [Build #306355](https://buildkite.com/elastic/kibana-pull-request/builds/306355) failed 75a535a742052bf4f71bbefa5c7d0da4516b99d9
* :broken_heart: [Build #306209](https://buildkite.com/elastic/kibana-pull-request/builds/306209) failed b775d25b8c700b5c1a9def9c44432d178a61e800

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306462","number":306462,"commit":"893601577dd3f1f35c7972ab4ed47c4aeffce039"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306428","number":306428,"commit":"582be89666777c946819d3ed5c3c5da5f3cdf692"},{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306413","number":306413,"commit":"570ddcfef71b2a20a401c37773dda7b04ed71d91"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306396","number":306396,"commit":"083fa1c326f332317b3a6acc0fbd6d7bd7703c33"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306355","number":306355,"commit":"75a535a742052bf4f71bbefa5c7d0da4516b99d9"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306209","number":306209,"commit":"b775d25b8c700b5c1a9def9c44432d178a61e800"}],"number":306462}
buildkite-pr-comment-->

legrego committed:
No-op 3

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/306612)
* Commit: 6760f14deb1f29c1e208033196cfb89068c0f5ab
* [Cloud Deployment](https://kibana-pr-222966.kb.us-west2.gcp.elastic-cloud.com)

### Failed CI Steps
* [FTR Configs #51](https://buildkite.com/elastic/kibana-pull-request/builds/306612#019754ec-8da1-41f2-900c-a74b6466ee06)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/306612#019754ec-8da1-41f2-900c-a74b6466ee06) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/306612/jobs/019754ec-8da1-41f2-900c-a74b6466ee06/artifacts/01975504-edda-49cd-88a4-b3734a2041c1) FTR Configs #<span></span>51 / Actions and Triggers app create alert should delete the right action when the same action has been added twice

### History
* :broken_heart: [Build #306462](https://buildkite.com/elastic/kibana-pull-request/builds/306462) failed 893601577dd3f1f35c7972ab4ed47c4aeffce039
* :broken_heart: [Build #306428](https://buildkite.com/elastic/kibana-pull-request/builds/306428) failed 582be89666777c946819d3ed5c3c5da5f3cdf692
* :broken_heart: [Build #306413](https://buildkite.com/elastic/kibana-pull-request/builds/306413) failed 570ddcfef71b2a20a401c37773dda7b04ed71d91
* :broken_heart: [Build #306396](https://buildkite.com/elastic/kibana-pull-request/builds/306396) failed 083fa1c326f332317b3a6acc0fbd6d7bd7703c33
* :broken_heart: [Build #306355](https://buildkite.com/elastic/kibana-pull-request/builds/306355) failed 75a535a742052bf4f71bbefa5c7d0da4516b99d9

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306612","number":306612,"commit":"6760f14deb1f29c1e208033196cfb89068c0f5ab"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306462","number":306462,"commit":"893601577dd3f1f35c7972ab4ed47c4aeffce039"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306428","number":306428,"commit":"582be89666777c946819d3ed5c3c5da5f3cdf692"},{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306413","number":306413,"commit":"570ddcfef71b2a20a401c37773dda7b04ed71d91"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306396","number":306396,"commit":"083fa1c326f332317b3a6acc0fbd6d7bd7703c33"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/306355","number":306355,"commit":"75a535a742052bf4f71bbefa5c7d0da4516b99d9"}],"number":306612}
buildkite-pr-comment-->

legrego committed:
Attempt to debug test failures

legrego committed:
more debugging

legrego committed:
Revert "more debugging"

This reverts commit bf135082a412eb31b101fd04898e7098bacf91b0.

legrego committed:
Revert "Attempt to debug test failures"

This reverts commit 6920f124c6a8fdf9b65806b0a37dfa7f0b485eb9.
